modules/utility/binds/which-key/which-key.nix

```diff
@@ -1,7 +1,77 @@
-{lib, ...}: let
-  inherit (lib) mkEnableOption;
+{
+  config,
+  lib,
+  ...
+}: let
+  inherit (lib) mkEnableOption mkOption mkOptionType types;
+
+  whichKeyCategory = mkOptionType {
+    name = "whichKeyCategory";
+    description = "The label to use for the which-key keybind helper menu.";
+    check = label:
+      if label == null && !(lib.isAttr label)
```

Comment:
To my knowledge, null comparisons (foo == null) are a no-op in nix. Use builtins.isNull

Reply:
I misremembered, nvm.

Reply:
The opposite is true. 😕